linter.py

```diff
@@ -40,19 +30,15 @@ class Lintr(Linter):
     line_col_base = (1, 1)
     tempfile_suffix = None
     error_stream = util.STREAM_BOTH
-    selectors = {}
     word_re = None
-    inline_settings = None
-    inline_overrides = None
-    comment_re = r'#'
     tempfile_suffix = 'lintr'
 
     def cmd(self):
         """Return a list with the command line to execute."""
-        settings = self.get_view_settings()
+        settings = self.settings
         command = 'library(lintr);lint(cache = {0}, commandArgs(TRUE), {1})'.format(settings['cache'],
                                                                                     settings['linters'])
-        return [self.executable_path,
```

Comment:
So does this mean R must be on the system `PATH`? I don't believe that was the case previously.

Reply:
Unfortunately yes, I'm not aware of any other solution

Reply:
It looks like the python plugin has a `python` option to set where python is located. (http://www.sublimelinter.com/en/stable/linter_settings.html#python) I think we should probably add an `r` option so you can do the same thing.

Reply:
Good idea! Will look into it